action.yml

```diff
@@ -113,6 +113,8 @@ runs:
             https://api.github.com/repos/${GITHUB_REPOSITORY}/code-scanning/sarifs \
             -d '{"commit_sha":"${GITHUB_SHA}","ref":"${GITHUB_REF}","sarif": "${COMPRESSED_SARIF}"}' || true
         fi
+    - name: Actions Tagger
+      uses: Actions-R-Us/actions-tagger@v2.0.2
```

Comment:
According to https://github.com/Actions-R-Us/actions-tagger#publish_latest_tag we would need to set this attribute - or not?
`
      - uses: Actions-R-Us/actions-tagger@latest
        with:
          publish_latest_tag: true
`

Reply:
Hi @berndruecker! That's correct, apologies. I updated the workflow in my latest PR to have the correct attributes as noted in the action-tagger's documentation.